.gitignore

```diff
@@ -22,3 +22,18 @@ backend/openapi.json
 .pytest_cache/
 
 /docker-compose.override.yml
+
+# PyCharm User-specific stuff:
+.idea/
```

Comment:
Wouldn't this line already be enough?

Reply:
Yes, if we just have `.idea/` the rest of the entries should be redundant